### PR TITLE
BITMAKER-2333: estela-web: Project & Spider deploy views issues

### PR DESCRIPTION
**Pull request by reqhiem**
# Description

-  estela need a welcome message for empty project list page: implemented

- Spider deploys needs to update to In Project/Spiders/Deploy: added welcome message to empty deploy list page

- Add a real demo project to deploy:  scrapy's quotesbot spider added

# Issue

[https://tasks.bitmaker.dev/issues/2333](https://tasks.bitmaker.dev/issues/2333)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] My code follows the style guidelines of this project.
- [x] I have made corresponding changes to the [documentation](https://github.com/bitmakerla/estela/tree/main/docs).
- [x] New and existing tests pass locally with my changes.
- [x] If this change is a core feature, I have added thorough tests.
- [x] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories. Please, refer to our [official documentation](https://estela.bitmaker.la/docs/).
- [x] I understand that my pull request may be closed if it becomes obvious or I did not perform all of the steps above.

### CODE REVIEW

File: estela-web/src/components/DeployListPage/index.tsx [hunk 1]
```diff
@@ -168,16 +258,16 @@ export class DeployListPage extends Component<RouteComponentProps<RouteParams>,
...
+                                        </Row>
                                         <Row className="my-6 grid grid-cols-11 text-base mx-10">
                                             <div className="col-span-5 flex flex-col font-sans text-xs mx-5">
                                                 <p className="font-medium text-center text-sm text-estela-black-full">
                                                     Deploy a demo
                                                 </p>
                                                 <div className="mt-4 rounded-md p-6 bg-back-code font-courier text-sm">
                                                     <p className=" text-white">
-                                                        $ github clone https://github.com/bitmaker/demospider.git
+                                                        $ github clone https://github.com/scrapy/quotesbot.git
```
Comment (JoseArmandoChavezQuijahuaman) [inline estela-web/src/components/DeployListPage/index.tsx:270]: Replace **github** with **git**

File: estela-web/src/components/ProjectListPage/index.tsx [hunk 2]
```diff
@@ -171,18 +180,60 @@ export class ProjectListPage extends Component<unknown, ProjectsPageState> {
...
+                                            <Text className="text-estela font-bold text-4xl">WELCOME SCRAPER!</Text>
+                                            <Paragraph className="text-xl mt-6">
+                                                Start by creating a <Text strong>project</Text> to be able to deploy
+                                                your spiders and start with your scraping.
+                                            </Paragraph>
+                                            <Paragraph className="text-lg font-bold">
+                                                Remember to install the&nbsp;
+                                                <a
+                                                    target="_blank"
+                                                    href="https://estela.bitmaker.la/docs/"
```
Comment (JoseArmandoChavezQuijahuaman) [inline estela-web/src/components/ProjectListPage/index.tsx:216]: I think the href is **https://estela.bitmaker.la/docs/estela-cli/install.html**

**Review by JoseArmandoChavezQuijahuaman — CHANGES_REQUESTED** [review]

File: estela-web/src/components/DeployListPage/index.tsx (hunk 1 above)
Comment (reqhiem) [reply estela-web/src/components/DeployListPage/index.tsx]: done.

File: estela-web/src/components/ProjectListPage/index.tsx (hunk 2 above)
Comment (reqhiem) [reply estela-web/src/components/ProjectListPage/index.tsx]: updated.

**Comment by reqhiem** [comment]
@JoseArmandoChavezQuijahuaman ready to review.

File: estela-web/src/components/DeployListPage/index.tsx (hunk 1 above)
Comment (JoseArmandoChavezQuijahuaman) [reply estela-web/src/components/DeployListPage/index.tsx]: Please, resolve the conversation

**Review by JoseArmandoChavezQuijahuaman — APPROVED** [review]
Nice job!

**Pull request merged**